doc/cluster-management/olvm.md

```diff
@@ -50,8 +50,7 @@ resources and terminology on the client.
 * All the network interface devices should be the same, such as enp1s0.
 
 ## Restrictions
-* This is a developer release feature so you must be using the developer RPM for the Oracle Cloud Native Environment CLI.
-* The OLVM Cluster API Provider does not support DHCP, you must allocate a range of external IPs.
+* The OLVM Cluster API Provider does not support DHCP, you must allocate a contiguous range of external IPs.
```

Comment:
The "external IP" concept is specific to OCI VLANs.  It would be good to clarify this so that it works in other environments.  What is really necessary is that the range is within the subnet of the network that the VMs are attached to.

Reply:
I added info the terminology section, please see if that is OK.